src/categories/categories.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable, Scope } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { BaseService } from '@shared/services';
+import { ReturnModelType } from '@typegoose/typegoose';
+
+import { Category } from './models/category.entity';
+import { CategoryDto } from './models/dto/category.dto';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['CategoryDto' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5693791)